models/sipnet/R/write.configs.SIPNET.R

```diff
@@ -437,7 +447,26 @@ write.config.SIPNET <- function(defaults, trait.values, settings, run.id, inputs
     if ("leafGrowth" %in% pft.names) {
       param[which(param[, 1] == "leafGrowth"), 2] <- pft.traits[which(pft.names == "leafGrowth")]
     }
-  }  ## end loop over PFTS
+
+    #update LeafOnday and LeafOffDay
+    if (update_phenology){
```

Comment:
Is there a reason that update_phenology needs to be a global variable rather than being a setting in `settings$model$leaf_phenology` or something similar?